packages/engine/src/avatar/AvatarMovementSystem.ts

```diff
@@ -1,9 +1,17 @@
+import { Engine } from 'behave-graph'
+import { Vector3 } from 'three'
+
+import { createActionQueue, dispatchAction, removeActionQueue } from '@xrengine/hyperflux'
+
+import { EngineActions } from '../ecs/classes/EngineState'
 import { World } from '../ecs/classes/World'
-import { applyGamepadInput } from './functions/moveAvatar'
+import { EngineRenderer } from '../renderer/WebGLRendererSystem'
+import { applyAutopilotInput, applyGamepadInput } from './functions/moveAvatar'
 
 export default async function AvatarMovementSystem(world: World) {
   const execute = () => {
     applyGamepadInput(world.localClientEntity)
+    applyAutopilotInput(world.localClientEntity)
```

Comment:
rather than adding to AvatarMovementSystem, we should keep this modular and make a copy of this file that handles applyAutopilotInput, such that we can  use each system in isolation